### PR TITLE
V2.4-WIP

### PR DESCRIPTION
**Pull request by Duality-CDOO**
### What is the purpose of this pull request (PR)?
v2.4.0.0

### CODE REVIEW

**Review by Duality-CDOO — COMMENTED** [review]
utACK

File: src/privatesend.h [hunk 1]
```diff
@@ -11,22 +11,22 @@
...
-static const int PRIVATESEND_QUEUE_TIMEOUT          = 30;
-static const int PRIVATESEND_SIGNING_TIMEOUT        = 15;
+static const int PRIVATESEND_AUTO_TIMEOUT_MIN = 5;
+static const int PRIVATESEND_AUTO_TIMEOUT_MAX = 15;
+static const int PRIVATESEND_QUEUE_TIMEOUT = 30;
+static const int PRIVATESEND_SIGNING_TIMEOUT = 15;
 
 //! minimum peer version accepted by mixing pool
-static const int MIN_PRIVATESEND_PEER_PROTO_VERSION = 70600;
+static const int MIN_PRIVATESEND_PEER_PROTO_VERSION = 70900;
```
[the remark this answers is not in the document]
Comment (AmirAbrams) [reply src/privatesend.h]: Yes, I think we should increase it to 71000.

File: dynamic-qt.pro [hunk 2]
```diff
@@ -646,4 +645,11 @@ FORMS += \
     src/qt/forms/dynodelist.ui \
     src/qt/forms/transactiondescdialog.ui \
     src/qt/forms/modaloverlay.ui \
-    src/qt/forms/miningpage.ui
+    src/qt/forms/miningpage.ui \
```
[the remark this answers is not in the document]
Comment (AmirAbrams) [reply dynamic-qt.pro]: Thanks, see https://github.com/duality-solutions/Dynamic/pull/228/commits/3d866b0d0f505820e0ed8e531ec737514784bae0

**Review by SvenKercher — APPROVED** [review]
tACK

**Pull request merged**